core/src/main/java/io/questdb/griffin/engine/table/DataFrameRecordCursor.java

```diff
@@ -53,6 +50,10 @@ public DataFrameRecordCursor(
         this.rowCursorFactory = rowCursorFactory;
         this.entityCursor = entityCursor;
         this.filter = filter;
+    }    private final BooleanSupplier nextRow = this::nextRow;
```

Comment:
intellij formatter breaks this file

Reply:
i see you changed the file, thanks! However pls also respond to comment and perhaps copy commit that fixed this. Otherwise I can only guess if the comment is addressed, which slows down the process.

Reply:
Fixed in 2f5e8c44 .